Aufg6/WieseIdee.ts

```diff
@@ -94,8 +95,8 @@ namespace Aufg6_Wiese {
 
     }
 
-    function biene(bees[0], bees[1], bees[2], bees[3]): void {
-
+    function biene(bees[0], bees[1], bees[2], bees[3]): void { <-- das ist eine unzulässige Form für eine Funktionsdefinition!
+        // auch hier mit dem Vorlesungscode vergleichen!

```

Comment:
Also die Biene soll sich selbst nur erstellen. Das müsste sie mit dem momentanen Code auch machen. Zusätzlich soll sie in der animate funktion 10 mal erstellt werden und jeweils andere x  und y. werte haben. 
Diese soll es dann berechnen und in den Array setzen, damit er sie dann aufrufen kann und hier scheitert es dann. 

"        for (let i: number = 0; i < n; i++) {
            let b: Bee = bees[i];
            b.x += Math.random() * 4 - 2;
            b.y += Math.random() * 4 - 2;
            b.x--;
            biene(b.x, b.y, 10, b.farbe);
"